web/src/app/projects/repository/repository.component.ts

```diff
@@ -31,10 +31,13 @@ export class RepositoryComponent implements OnInit {
       .subscribe((repository: RepositoryModel) => {
         this.repository = repository;
         if (this.repository.milestones) {
-          this.sortingService.sortList(this.repository.milestones, 'updatedAt');
+          this.sortingService.sortListByDate(this.repository.milestones, 'updatedAt');
```

Comment:
Generic type needs to be provided to the method, is there a tslint option for this?

```typescript
this.sortingService.sortListByDate<MilestoneModel>(this.repository.milestones, 'updatedAt');
```

Reply:
"callable-types" : true . 
This configuration is not giving any error

Reply:
Lets leave it for now, I think in another PR we should put the linter to strict on everything